examples/dynamo/eg3/solver_mod.x90

```diff
@@ -450,7 +450,7 @@ contains
 
     !PSY call invoke ( minus_field_data(rhs,Ax,res))
     call invoke_minus_field_data( rhs, Ax, res )
-    !PSY call invoke ( divide_field(res, lumped_weight, res))
+    !PSY call invoke ( divide_fields(res, lumped_weight, res))
```

Comment:
See the comment above about replacing PSy-Kal lite with PSyclone invokes. There's enough work for a separate issue.